bin/quest

```diff
@@ -65,6 +65,22 @@ module GLIWrapper
     output << "#{complete_count} of #{summary['example_count']} tasks."
   end
 
+  def wait_spinner
```

Comment:
In this case it seems simpler to just add it myself than add a dependency. It looks like that gem is intended to react to an event that takes place within the block, whereas I just want something on an independent timer.